### DISCUSSION TITLE
Unable to Toggle Line Comment

### DISCUSSION THREAD

**Discussion by CtfChan** [coder/code-server#2198] [Q&A]
Hi guys, 

I'm using Chrome with code-server and I am unable to to comment code using `control` + `/` . Has anyone ever experienced a similar issue?

Cheers.

**Comment by code-asher** [comment]
I tested it out on 3.5.0 and 3.6.0 with Chromium (version 85.0.4183.121)
and it's working for me when I'm editing a file that isn't plain text.

Maybe it's bound to something else? Could be worth a shot to open the
keyboard shortcuts and double-check.
